ansible/roles/openshift_template_deployer/handlers/main.yml

```diff
@@ -0,0 +1,10 @@
+---
+- name: restart openshift master services
+  service:
+    name: "{{ item[1] }}"
+    state: restarted
+  delegate_to: "{{ item[0] }}"
+  with_nested:
+  - "{{ osvp_master_nodes }}"
```

Comment:
We all waivered on just removing the handler, but I'll update the variable just in case we decide to keep this file.